etl-federation/existing-prometheus/README.md

```diff
@@ -0,0 +1,165 @@
+# Overview
+
+These are the steps to enable ETL federation with Kubecost.
+
+When using an existing prometheus instance, Kubecost only requires a single pod per cluster. The example here also enables the networkCosts Daemonset which is optional.
+
+Contact us for help customizing settings.
```

Comment:
I recommend to add a hyperlink here:
Example:

[Contact us](https://www.kubecost.com/contact) for help customizing settings.